soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -3947,14 +3945,17 @@ void DrawRandoEditor(bool& open) {
                 ImGui::PushItemWidth(-FLT_MIN);
 
                 //Starting Age
-                //Disabled when Forest is set to Closed
-                bool disableRandoStartingAge = !CVar_GetS32("gRandomizeForest", 0);
-                const char* disableRandoStartingAgeText = "This option is disabled because \"Forest\" is set to \"Closed\".";
+                //Disabled when Forest is set to Closed or under very specific conditions
+                bool disableRandoStartingAge = !CVar_GetS32("gRandomizeForest", 0) || //If Forest is closed...
```

Comment:
I haven't been keeping up to date with the rando chats in Discord lately, but why would Closed Forest cause an unbeatable seed with random starting age? Or is it not unbeatable, just wildly out of logic?

Reply:
Closed Forest and Adult are not compatible, according to 3D logic. (To access Deku Tree as intended, you'd need both the Kokiri sword and the Deku Shield, which Adult can't get even through noclip since KF has the sword chest unloaded in his map.) I'm guessing that Random is similarly disabled due to this possibility being less than certain, but not zero either.

Reply:
Oh does it not start you in the temple of time if you start out as Adult? I assumed changing the starting Age would also change where you spawn, but it sounds like that's not the case?

Reply:
Changing ages does place you somewhere else (which had to be implemented on the SoH side), but then the problem becomes you being sort of softlocked once you change back to child, which would likely happen since you can grab more checks in the beginning as child rather than as adult logically.

Reply:
Oh yeah I guess if logic accounts for you starting outside of the Forest as Child and places items needed for Closed Forest out there, then you would softlock if you save and reset before getting them. And the alternative is keeping all the items needed for Deku Tree accessible from within Kokiri Forest which would be pretty boring.